packages/flutter/test/widgets/orientation_builder_test.dart

```diff
@@ -0,0 +1,253 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('OrientationBuilder', () {
```

Comment:
I can't believe we didn't previously have any tests for OrientationBuilder! Thanks for including some here.